### DISCUSSION TITLE
Which parameter is most important in input.json file?

### DISCUSSION THREAD

**Discussion by hanrm94** [deepmodeling/deepmd-kit#3368] [Q&A]
In the input.json run by DeepMD, what parameters can be modified to make the trained force field more accurate?

**Comment by njzjz** [comment]
I think it's the number of the training steps, the loss function, and the learning rate.
